### PR TITLE
Add json schema for IRIS Ask a Question

### PR DESCRIPTION
**Pull request by shefalinayak**
# New schema

department-of-veterans-affairs/orchid#17

This schema is for the new IRIS Ask a Question form (replacing https://iris.custhelp.va.gov/app/ask) that will live on VA.gov.

## Pull Requests to update the schema in related repositories

vets-website PR to be created
vets-api PR to be created

### CODE REVIEW

**Comment by stephenBarrs** [comment]
Hi! Just FYI in case you do not know, when adding a new Schema to `vets-json-schema` you have to add a test to `vets-website` or to an exclusion list or the `vets-website` tests will break (I found this out the difficult way!).  Here is the exclusion list: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms/tests/forms.unit.spec.js#L71

**Comment by shefalinayak** [comment]
> Hi! Just FYI in case you do not know, when adding a new Schema to `vets-json-schema` you have to add a test to `vets-website` or to an exclusion list or the `vets-website` tests will break (I found this out the difficult way!). Here is the exclusion list: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms/tests/forms.unit.spec.js#L71

@stephenBarrs I did not know, thank you for the info. Created a [vets-website PR](https://github.com/department-of-veterans-affairs/vets-website/pull/13808) to add the form to the exclusion list.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.6.3",
+  "version": "6.7.3",
```
Comment (f1337) [inline package.json:3]:
```suggestion
  "version": "6.7.0",
```

Per [semver](https://semver.org), shouldn't this be `6.7.0`?

Comment (shefalinayak) [reply package.json]: Right, sorry we missed that!

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.6.3",
+  "version": "6.7.0",
```
Comment (f1337) [inline package.json:3]: Please double-check the version in `master` before merging? There are several PRs open right now. Thank you!!

**Pull request merged**